apps/expo/src/app/privacy-policy/index.tsx

```diff
@@ -0,0 +1,225 @@
+            >
+              1. Information We Collect
+            </Text>
+
+            <Text className='text-2xl text-slate-200' style={{ paddingHorizontal: 10 }}>
+              {'\n'}a. Personal Information: {'\n\t\t\t'} When you create an account with Barbell,
+              we may collect personal information such as your name, email address, age, gender, and
+              profile picture. Additionally, if you choose to link your Spotify account, we may
+              collect information from Spotify's API, including your current listening activity.
+              {'\n\n'}b. Workout Data:{'\n\t\t\t'} Barbell collects information about your workout
+              activities, including exercise routines, duration, intensity, and progress. This data
+              may be shared with friends and the public by default within the App.{'\n\n'}c. Device
+              Information:
+              {'\n\t\t\t'} We may collect information about your device, including its unique device
+              identifier, operating system, and mobile network information.{'\n\n'}d. Location
+              Information:{'\n\t\t\t'} With your consent, we may collect and process information
+              about your precise or approximate location when you use certain features of the App,
+              such as mapping your running routes.{'\n\n'}
+            </Text>
+          </View>
+
+          <View className={tailwindClasses.mainTile}>
+            <Text
+              className='text-2xl text-slate-200'
+              style={{ fontSize: 25, paddingHorizontal: 10 }}
+            >
+              2. How We Use Your Information
+            </Text>
+            <Text className='text-2xl text-slate-200' style={{ paddingHorizontal: 10 }}>
+              {'\n'}a. Personalization:{'\n\t\t\t'} We use the information collected to personalize
+              your experience with the App, including suggesting workout routines, tracking
+              progress, and providing relevant content.{'\n\n'}b. Social Sharing:{'\n\t\t\t'} Your
+              workout data may be shared with friends and the public within the App. However, you
+              have control over the visibility settings of your profile and can adjust the privacy
+              settings accordingly.
+              {'\n\n'}c. Integration with Spotify:{'\n\t\t\t'} Integration with Spotify: If you
+              choose to link your Spotify account, we may access your Spotify data through Spotify's
+              API to display your current listening activity to others on the App. This feature
+              enhances social interaction within the fitness community, and you may choose to opt
+              out at any time from user settings should you change your mind.{'\n\n'}d. Analytics:
+              {'\n\t\t\t'} We may use aggregated and anonymized data for analytical purposes to
+              improve the performance and functionality of the App.{'\n\n'}
```

Comment:
brother, why you have so many \n\n.

IMO, remove them

Reply:
You're right, I removed the '\n\n' at the end of all the sections